scripts/dot-properties/main.py

```diff
@@ -15,16 +16,10 @@ def main():
     config = ConfigParser()
     config.read(CONFIG_FILE)
 
-    aws_profile_name = _default_option(config, 'aws', 'profile-name', 'media-service')
-    aws_region = _default_option(config, 'aws', 'region', 'eu-west-1')
-    cf_stack = _default_option(config, 'aws', 'stack-name', None)
-
-    if not cf_stack:
-        raise Exception('No stack found in {}'.format(CONFIG_FILE))
```

Comment:
`cf_stack` is now calculated in a similar way to https://github.com/guardian/grid/blob/master/cloud-formation/scripts/stack-name.sh